scripts/examples/gn_silabs_example.sh

```diff
@@ -214,6 +214,10 @@ else
             #   optArgs+="openthread_root=\"//third_party/connectedhomeip/third_party/openthread/ot-efr32/openthread\" openthread_efr32_root=\"//third_party/connectedhomeip/third_party/openthread/ot-efr32/src/src\""
             #    shift
             #    ;;
+            --report_on_active)
+                optArgs+="chip_report_on_active_mode=true "
+                shift
+                ;;
```

Comment:
I don't think we need a macro for this argument. This is going to have a default config for our LIT ICD examples.
@jmartinez-silabs agree?

Reply:
removed

Reply:
yeah I agree, no macro/preset for 1 arg, and if we want to enable it by default when LIT is supported we can set it in the example gni